helm_deploy/values-base.yaml

```
@@ -159,6 +159,8 @@ elasticsearchSetupJob:
     capabilities:
       drop: ["ALL"]
       add: ["NET_BIND_SERVICE"]
+    # Schedule of CronJob when enabled
+    schedule: "0 8 * * *"
```

I'd recommend just using weekdays unless it's prod, as most environments will have shut down the aws resources over the weekend, which could cause issues.

```suggestion
    schedule: "0 8 * * 1-5"
```